pkgs/development/python-modules/nimbo/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, python3, awscli }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "nimbo";
+  version = "v0.2.1";
+
+  disabled = python3.pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "nimbo-sh";
+    repo = pname;
+    sha256 = "0qagib2607qvp5fl71325qpyrpvfhgmk1zix0nbws91n6i3fzc6i";
+    rev = version;
```

Comment:
```suggestion
  version = "0.2.1";

  disabled = python3.pythonOlder "3.6";

  src = fetchFromGitHub {
    owner = "nimbo-sh";
    repo = pname;
    sha256 = "0qagib2607qvp5fl71325qpyrpvfhgmk1zix0nbws91n6i3fzc6i";
    rev = "$v{version}";
```

Reply:
fixed in b6fca896de9f4b08d8b729c813be03758e8113df